lib/fog/ovirt/compute/v4.rb

```diff
@@ -1,3 +1,5 @@
+require "active_support"
```

Comment:
`active_support` is part of rails, do you have to use it here? I think Fog does not require it.

Reply:
I still don't see a reason to require `active_support` here.
If you need it for testing, it would be better to require it in test helpers. Although if the code should run without `active_support`, we should run the tests without it too. Otherwise we can slip some `active_suport` stuff in our code, and it won't be detected until the code is executed in prod setup.